services/core/java/com/android/server/lights/LightsService.java

```diff
@@ -88,7 +96,11 @@ public void setBrightness(int brightness, int brightnessMode) {
                             (float) brightness / mSurfaceControlMaximumBrightness);
                 } else {
                     int color = brightness & 0x000000ff;
-                    color = 0xff000000 | (color << 16) | (color << 8) | color;
+                    if (mUseScaleBrightness > 0) {
```

Comment:
So the value of `0` also disables. Intentional?

Reply:
> So the value of `0` also disables. Intentional?

who will set their scaled brightness from 0 to 0 anyways ?

Reply:
That's true. Doesn't make a real difference anyways. Could use `0` as default than, but again, doesn't really matter